src/initializers/koa/routes.ts

```diff
@@ -0,0 +1,23 @@
+import * as os from 'os';
+
+export default {
+  get: {
+    '/health': async ctx => (ctx.status = 200),
```

Comment:
Since orka knows if you use: kafka, redis, mongo or rabbit we could add those connections in the health endpoint. If the connections are not healthy we should not return 200.

Reply:
I'm not opposed. I believe it's a good practice and I believe some teams are already doing this.
I'm troubled by the following though:
1. It's very intrusive and some teams might not want it (this can be solved by configuration though)
2. These checks are not "free", meaning that health checks should have a reasonable interval (check with sre team)